utils/build.ps1

```diff
@@ -1963,7 +1963,8 @@ function Build-LLVM([Hashtable] $Platform) {
 function Build-Sanitizers([Hashtable] $Platform) {
   $LLVMTargetCache = $(Get-ProjectBinaryCache $Platform LLVM)
   $LITVersionStr = $(Invoke-Program $(Get-PythonExecutable) "$LLVMTargetCache\bin\llvm-lit.py" --version)
-  if (-not ($LITVersionStr -match "lit (\d+)\.\d+\.\d+.*")) {
+  if (-not $ToBatch -and -not ($LITVersionStr -match "lit (\d+)\.\d+\.\d+.*")) {
+    Write-Host "$LITVersionStr" -ForegroundColor Red
```

Comment:
Perhaps consider merging that into the thrown error instead?

Reply:
I'm not sure why I didn't do that. I think I'd just added this while debugging the `$ToBatch` stuff (which I was only using so I could extract a particular CMake invocation).

Reply:
Yeah, there is another set of changes that @jeffdav had started which would be nice to finish up and merge (`-WhatIf` support). For now, I think that we should just merge this into the error message.

Reply:
Done.